feat: add Spanish translation for the audit script

macaron-jpg opened the pull request:
since I was nervous and new to this I did this entirely with GitHub's GUI, let me know if you find any issues (:

macaron-jpg committed:
added folder and file with spanish translation

since I was nervous and new to this I did this entirely with GitHub's GUI, let me know if you find any issues (:

macaron-jpg committed:
Update audit_secureblue.po

fixed typo

RoyalOughtness commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:165
+msgid "Ensuring no sysctl overrides"
+msgstr "Asegurando de que no haya sobreescritura de métodos de sysctl"
```
what do you mean by sysctl methods?

The english is potentially too vague here, but the intended meaning is akin to "ensuring no overrides to sysctl values"

maybe "valores de sysctl"?

macaron-jpg replied:
I've updated the file c:

macaron-jpg committed:
Update audit_secureblue.po

changes:
-"esté/este en uso/etc." to "está en uso/etc."
-line 484 "remoto de flatpaks" to "repositorio remoto de flatpaks" https://github.com/flatpak/flatpak/blob/f53cef00417955eebb0f3fecab82270097e2be15/po/es.po#L1624
-"asegurándose de que no haya sobrescritura de métodos de x" to "asegurándose de que no haya sobrescritura de x"
-"del repositorio" to "desde el repositorio"
-"MAC aleatorio" to "aleatorización de MAC" https://source.android.com/docs/core/connect/wifi-mac-randomization-behavior?hl=es-419
-"usuario común" to "usuario no root" https://www.ibm.com/docs/es/capm?topic=agents-installing-as-non-root-user
-"aplicaciones flatpak" to "flatpak(s)"
-"Para remover este permiso de un flatpak" to "Para removerles este permiso"
-"de la sandbox" (which should've been "del sandbox") and "sandbox" in general to "entorno aislado" https://github.com/flatpak/flatpak/blob/f53cef00417955eebb0f3fecab82270097e2be15/po/es.po#L2362
-"ejecutar" to "ejecute"; except for line 764 and a portion of line 83

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

macaron-jpg committed:
Update audit_secureblue.po

changed:
-keep hardened_malloc untranslated

macaron-jpg committed:
Update audit_secureblue.po

-updated revision date

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:329
+msgid "To set up USBGuard, run:"
+msgstr "Para activar USBGuard, ejecute:"
```
```suggestion
msgstr "Para configurar USBGuard, ejecute:"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:671
+msgid "To set it to Enforcing mode, run:"
+msgstr "Para ponerlo en Modo Reforzado, ejecute:"
```
```suggestion
msgstr "Para configurarlo en Modo Reforzado, ejecute:"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:698
+msgid "To reset it, run:"
+msgstr "Para reiniciarlo, ejecute:"
```
```suggestion
msgstr "Para restablecerlo, ejecute:"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:745
+#, python-brace-format
+msgid "The file {0} was not found."
+msgstr "El archivo {0} no fue hallado."
```
```suggestion
msgstr "No se encontró el archivo {0}."
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:760
+msgid "To reset it and enable hardened_malloc for system processes, run:"
+msgstr "Para reiniciarlo y habilitar hardened_malloc para procesos del sistema, ejecute:"
```
```suggestion
msgstr "Para restablecer y habilitar hardened_malloc para procesos del sistema, ejecute:"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:783
+#, python-brace-format
+msgid "{0} is set, but {1} has been modified."
+msgstr "{0} está colocado, pero {1} ha sido modificado."
```
```suggestion
msgstr "{0} está configurado, pero {1} ha sido modificado."
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:791
+#, python-brace-format
+msgid "The '{0}' variant of {1} has been set."
+msgstr "El variante '{0}' de {1} se ha colocado."
```
```suggestion
msgstr "La variante '{0}' de {1} ha sido establecida."
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:794
+#, python-brace-format
+msgid "{0} has not been set."
+msgstr "{0} no se ha colocado."
```
```suggestion
msgstr "{0} no se ha establecido."
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+"                Check that {1} has not been overridden in\n"
+"                {2} or related configuration files."
+msgstr ""
+"La variable de entorno {0} ha sido modificada o no está colocada.\n"
```
```suggestion
"La variable de entorno {0} ha sido modificada o no está establecida.\n"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_flatpak/__init__.py:376
+#: files/system/usr/libexec/secureblue/audit_flatpak/__init__.py:415
+msgid "To remove this permission from an app, use Flatseal or run:"
+msgstr "Para removerles este permiso, use Flatseal o ejecute:"
```
```suggestion
msgstr "Para eliminar este permiso de una aplicación, use Flatseal o ejecute:"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_flatpak/__init__.py:193
+msgid "This grants access to audio and microphones."
+msgstr "Esto otorga acceso al audio y microphono."
```
```suggestion
msgstr "Esto otorga acceso al audio y micrófonos."
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_flatpak/__init__.py:195
+msgid "access to the D-Bus system bus"
+msgstr "acesso al bus del sistema D-Bus"
```
```suggestion
msgstr "acceso al bus del sistema D-Bus"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_flatpak/__init__.py:203
+#, python-brace-format
+msgid "If GPU access is required, allow {0} instead."
+msgstr "Si acceso al GPU es requerido, permita {0} en su lugar."
```
```suggestion
msgstr "Si se requiere acceso a la GPU, permita {0} en su lugar."
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_flatpak/__init__.py:266
+msgid "However, this is required for its functionality."
+msgstr "Pero esto es requerido para su funcionalidad."
```
```suggestion
msgstr "Sin embargo, esto es necesario para su funcionamiento."
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_flatpak/__init__.py:396
+msgid "To add this permission to an app, use Flatseal or run:"
+msgstr "Para añadir este permiso a un flatpak, use Flatseal o ejecute:"
```
```suggestion
msgstr "Para añadir este permiso a una aplicación, use Flatseal o ejecute:"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/utils/__init__.py:104
+msgid "no app permissions flagged (however, not all permissions are audited)."
+msgstr "ningún permiso de aplicación fue señalado (siendo dicho, no todos los permisos son verificados)."
```
```suggestion
msgstr "ningún permiso de aplicación fue señalado (sin embargo, no todos los permisos son verificados)."
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:718
+#, python-brace-format
+msgid "The file {0} was not found or inaccessible."
+msgstr "El archivo {0} no fue hallado o es inaccesible."
```
```suggestion
msgstr "No se encontró el archivo {0} o es inaccesible."
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:851
+msgid "To fix this, run:"
+msgstr "Para arreglar esto, ejecute:"
```
```suggestion
msgstr "Para solucionar esto, ejecute:"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+"            to certain directories, direct D-Bus access, X11)."
+msgstr ""
+"la aplicación tiene permisos que pueden ser usados para escapar del entorno aislado, permitirle\n"
+"            que modifique sus propios permisos, o otorgarle acceso muy amplio al sistema "
```
```suggestion
"            que modifique sus propios permisos, u otorgarle acceso muy amplio al sistema "
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+"            weaken security (e.g. PulseAudio, Bluetooth, not using "
+"hardened_malloc)."
+msgstr ""
+"la aplicación tiene permisos que tienen potencial para ser usados para escapar del entorno aislado o\n"
```
```suggestion
"la aplicación tiene permisos con potencial para ser usados para escapar del entorno aislado o\n"
```

radargt reviewed (commented):
Algunas correciones.

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Para activar USBGuard, ejecute:" to 
"Para configurar USBGuard, ejecute:"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Para ponerlo en Modo Reforzado, ejecute:" to "Para configurarlo en Modo Reforzado, ejecute:"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Para reiniciarlo, ejecute:" to "Para restablecerlo, ejecute:"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Para reiniciarlo, ejecute:" to "Para restablecerlo, ejecute:"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Para reiniciarlo y habilitar hardened_malloc para procesos del sistema, ejecute:" to  "Para restablecer y habilitar hardened_malloc para procesos del sistema, ejecute:"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Para reiniciarlo y habilitar hardened_malloc para procesos del sistema, ejecute:" to  "Para restablecer y habilitar hardened_malloc para procesos del sistema, ejecute:"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"El archivo {0} no fue hallado o es inaccesible." to "No se encontró el archivo {0} o es inaccesible."

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Para arreglar esto, ejecute:" to "Para solucionar esto, ejecute:"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"que modifique sus propios permisos, o otorgarle acceso muy amplio al sistema " to "que modifique sus propios permisos, u otorgarle acceso muy amplio al sistema "

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"la aplicación tiene permisos que tienen potencial para ser usados para escapar del entorno aislado o\n" to
"la aplicación tiene permisos con potencial para ser usados para escapar del entorno aislado o\n"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update audit_secureblue.po

-changed revision date

macaron-jpg committed:
Update audit_secureblue.po

-typo fixes

macaron-jpg committed:
Update audit_secureblue.po

-"haya sobreescritura" to "hayan sobreescrituras"
-lines 242 & 254 "habilitado" to  "habilitada"

macaron-jpg committed:
Update audit_secureblue.po

-fixed revision date syntax

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

RoyalOughtness committed:
Merge branch 'live' into spanish-translation

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que chronyd está activo" to "Asegurando que chronyd está activo"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Creación del dominio de espacios de nombres de usuarios para el uso de contenedores está permitido." to "Creación del dominio de espacios de nombres de usuarios para el uso de contenedores está permitida."

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que la creación de espacios de nombres de usuarios no confinados está prohibido" to "Asegurando de que la creación de espacios de nombres de usuarios no confinados está prohibida"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Para más información sobre lo que significa, vea:" to  "Para más información sobre lo que esto significa, vea:"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Argumento del kernel faltante: {0} (soporte 32-bit)" to  "Argumento del kernel faltante: {0} (soporte de 32 bits)"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Los archivos siguientes no parecen ser inmutables o no existen:" to  "Los siguientes archivos no parecen ser inmutables o no existen:"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Los siguientes flatpaks pueden modificar la sobreescritura de métodos de flatpaks:" to "Los siguientes flatpaks pueden modificar los permisos de otros flatpaks:"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_flatpak/__init__.py:155
+msgid "This may also be used as a sandbox escape vector."
+msgstr "Este puede ser usado como vector de escape del entorno aislado."
```
Here they use "entorno aislado": https://github.com/flatpak/flatpak/blob/f53cef00417955eebb0f3fecab82270097e2be15/po/es.po#L1469
What do you think?

macaron-jpg replied:
Oh my god haha

macaron-jpg replied:
Yeah, I've noticed that they have some inconsistencies sometimes

macaron-jpg replied:
I'll change it to "sandbox" for now c:

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Las variables de entorno de Bash no está confinadas." to "Las variables de entorno de Bash no están confinadas."

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"{0} está colocado, pero {1} ha sido modificado."
 to "{0} está configurado, pero {1} ha sido modificado."

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"El variante '{0}' de {1} se ha colocado." to "La variante '{0}' de {1} ha sido establecida."

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"{0} no se ha colocado." to "{0} no se ha establecido."

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"La variable de entorno {0} ha sido modificada o no está colocada.\n" to "La variable de entorno {0} ha sido modificada o no está establecida.\n"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Para removerles este permiso, use Flatseal o ejecute:" to "Para eliminar este permiso de una aplicación, use Flatseal o ejecute:"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Esto otorga acceso al audio y microphono." to "Esto otorga acceso al audio y micrófonos."

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"acesso al bus del sistema D-Bus" to "acceso al bus del sistema D-Bus"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Si acceso al GPU es requerido, permita {0} en su lugar." to "Si se requiere acceso a la GPU, permita {0} en su lugar."

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Pero esto es requerido para su funcionalidad." to "Sin embargo, esto es necesario para su funcionamiento."

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Para añadir este permiso a un flatpak, use Flatseal o ejecute:" to "Para añadir este permiso a una aplicación, use Flatseal o ejecute:"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que el arranque seguro está habilitado" to "Asegurando que el arranque seguro está habilitado"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"entorno aislado" to "sandbox"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"entorno aislado" to "sandbox"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"vectores de escape del entorno aislado" to "posibles vectores de escape del sandbox"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"entorno aislado" to "sandbox"

Co-authored-by: Collision4468 <204792006+Collision4468@users.noreply.github.com>

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

macaron-jpg committed:
Update audit_secureblue.po

-added new revision date

macaron-jpg reviewed (commented):
c:

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:165
+msgid "Ensuring no sysctl overrides"
+msgstr "Asegurando de que no hayan sobreescrituras de sysctl"
```
```suggestion
msgstr "Asegurando que no hayan sobreescrituras de sysctl"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:182
+msgid "Ensuring a signed image is in use"
+msgstr "Asegurando de que una imagen firmada está en uso"
```
```suggestion
msgstr "Asegurando que una imagen firmada está en uso"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:215
+msgid "Ensuring no modprobe overrides"
+msgstr "Asegurando de que no hayan sobreescrituras de modprobe"
```
```suggestion
msgstr "Asegurando que no hayan sobreescrituras de modprobe"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:251
+msgid "Ensuring ptrace is forbidden"
+msgstr "Asegurando de que ptrace está prohibido"
```
```suggestion
msgstr "Asegurando que ptrace está prohibido"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:261
+msgid "Ensuring no authselect overrides"
+msgstr "Asegurando de que no hayan sobreescrituras de authselect"
```
```suggestion
msgstr "Asegurando que no hayan sobreescrituras de authselect"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:277
+msgid "Ensuring no container policy overrides"
+msgstr "Asegurando de que no hayan sobreescrituras de políticas de contenedores"
```
```suggestion
msgstr "Asegurando que no hayan sobreescrituras de políticas de contenedores"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:294
+msgid "Ensuring unconfined user namespace creation disallowed"
+msgstr "Asegurando de que la creación de espacios de nombres de usuarios no confinados está prohibida"
```
```suggestion
msgstr "Asegurando que la creación de espacios de nombres de usuarios no confinados está prohibida"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:311
+msgid "Ensuring container user namespace creation disallowed"
+msgstr "Asegurando de que la creación del dominio de espacios de nombres de usuarios para el uso de contenedores está prohibido"
```
```suggestion
msgstr "Asegurando que la creación del dominio de espacios de nombres de usuarios para el uso de contenedores está prohibida"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:336
+msgid "Ensuring USBGuard is active"
+msgstr "Asegurando de que USBGuard está activo"
```
```suggestion
msgstr "Asegurando que USBGuard está activo"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:403
+msgid "Ensuring system DNS resolution is secure"
+msgstr "Asegurando de que la resolución de conexión DNS sea segura"
```
```suggestion
msgstr "Asegurando que la resolución de conexión DNS sea segura"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:434
+msgid "Ensuring MAC randomization is enabled"
+msgstr "Asegurando de que la aleatorización de MAC está habilitada"
```
```suggestion
msgstr "Asegurando que la aleatorización de MAC está habilitada"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:579
+#, python-brace-format
+msgid "Ensuring {0} is enabled"
+msgstr "Asegurando de que {0} está habilitado"
```
```suggestion
msgstr "Asegurando que {0} está habilitado"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:554
+#, python-brace-format
+msgid "Ensuring {0} is enabled globally"
+msgstr "Asegurando de que {0} está habilitado globalmente"
```
```suggestion
msgstr "Asegurando que {0} está habilitado globalmente"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:600
+msgid "Ensuring user is not a member of the wheel group"
+msgstr "Asegurando de que el usuario no sea miembro del grupo wheel"
```
```suggestion
msgstr "Asegurando que el usuario no sea miembro del grupo wheel"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:630
+#, python-brace-format
+msgid "Ensuring {0} is disabled for {1}"
+msgstr "Asegurando de que {0} está deshabilitado para {1}"
```
```suggestion
msgstr "Asegurando que {0} está deshabilitado para {1}"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:658
+msgid "Ensuring GNOME user extensions are disabled"
+msgstr "Asegurando de que las extensiones de usuarios de GNOME están deshabilitadas"
```
```suggestion
msgstr "Asegurando que las extensiones de usuarios de GNOME están deshabilitadas"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:675
+msgid "Ensuring SELinux is in Enforcing mode"
+msgstr "Asegurando de que SELinux está en Modo Reforzado"
```
```suggestion
msgstr "Asegurando que SELinux está en Modo Reforzado"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:702
+msgid "Ensuring no environment file overrides"
+msgstr "Asegurando de que no hayan sobreescrituras de variables de entorno de archivos"
```
```suggestion
msgstr "Asegurando que no hayan sobreescrituras de variables de entorno de archivos"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:731
+msgid "Ensuring KDE GHNS is disabled"
+msgstr "Asegurando de que KDE GHNS está deshabilitado"
```
```suggestion
msgstr "Asegurando que KDE GHNS está deshabilitado"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:765
+#, python-brace-format
+msgid "Ensuring {0} has expected permissions"
+msgstr "Asegurando de que {0} tenga los permisos esperados"
```
```suggestion
msgstr "Asegurando que {0} tenga los permisos esperados"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+"                {2} o en archivos de configuración relacionados."
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:803
+msgid "Ensuring hardened_malloc is set to be preloaded"
+msgstr "Asegurando de que hardened_malloc está listo para ser precargado"
```
```suggestion
msgstr "Asegurando que hardened_malloc está listo para ser precargado"
```

radargt commented on files/po/es/audit_secureblue.po:
```diff
@@ -0,0 +1,842 @@
+
+#: files/system/usr/libexec/secureblue/audit_secureblue.py:858
+msgid "Ensuring current user's bash environment is locked down"
+msgstr "Asegurando de que las variables de entorno de Bash del usuario están confinadas"
```
```suggestion
msgstr "Asegurando que las variables de entorno de Bash del usuario están confinadas"
```

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que no hayan sobreescrituras de sysctl" to "Asegurando que no hayan sobreescrituras de sysctl"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que una imagen firmada está en uso" to "Asegurando que una imagen firmada está en uso"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que no hayan sobreescrituras de modprobe" to "Asegurando que no hayan sobreescrituras de modprobe"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que ptrace está prohibido" to "Asegurando que ptrace está prohibido"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que no hayan sobreescrituras de variables de entorno de archivos" to "Asegurando que no hayan sobreescrituras de variables de entorno de archivos"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que no hayan sobreescrituras de authselect" to "Asegurando que no hayan sobreescrituras de authselect"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que KDE GHNS está deshabilitado" to "Asegurando que KDE GHNS está deshabilitado"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que {0} tenga los permisos esperados" to "Asegurando que {0} tenga los permisos esperados"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que no hayan sobreescrituras de políticas de contenedores" to "Asegurando que no hayan sobreescrituras de políticas de contenedores"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que la creación de espacios de nombres de usuarios no confinados está prohibida" to "Asegurando que la creación de espacios de nombres de usuarios no confinados está prohibida"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que hardened_malloc está listo para ser precargado" to "Asegurando que hardened_malloc está listo para ser precargado"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que las variables de entorno de Bash del usuario están confinadas" to "Asegurando que las variables de entorno de Bash del usuario están confinadas"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que la creación del dominio de espacios de nombres de usuarios para el uso de contenedores está prohibido" to "Asegurando que la creación del dominio de espacios de nombres de usuarios para el uso de contenedores está prohibida"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que USBGuard está activo" to "Asegurando que USBGuard está activo"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que la resolución de conexión DNS sea segura" to "Asegurando que la resolución de conexión DNS sea segura"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que la aleatorización de MAC está habilitada" to "Asegurando que la aleatorización de MAC está habilitada"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que {0} está habilitado" to "Asegurando que {0} está habilitado"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que {0} está habilitado globalmente" to  "Asegurando que {0} está habilitado globalmente"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que el usuario no sea miembro del grupo wheel" to "Asegurando que el usuario no sea miembro del grupo wheel"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que {0} está deshabilitado para {1}" to  "Asegurando que {0} está deshabilitado para {1}"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que las extensiones de usuarios de GNOME están deshabilitadas" to "Asegurando que las extensiones de usuarios de GNOME están deshabilitadas"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update files/po/es/audit_secureblue.po

"Asegurando de que SELinux está en Modo Reforzado" to  "Asegurando que SELinux está en Modo Reforzado"

Co-authored-by: Radar <197807510+radargt@users.noreply.github.com>

macaron-jpg committed:
Update audit_secureblue.po

-changed revision date

RoyalOughtness committed:
Merge branch 'live' into spanish-translation

macaron-jpg committed:
Delete files/po/es directory

-deleted to replace with fixed ver

macaron-jpg committed:
feat: add spanish translation for audit script

fixed

macaron-jpg committed:
Merge branch 'live' into spanish-translation

macaron-jpg committed:
Update audit_secureblue.po

-fixed revision date

macaron-jpg committed:
Update audit_secureblue.po

-should've been 17 not 18

RoyalOughtness committed:
Merge branch 'live' into spanish-translation

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

macaron-jpg committed:
Merge branch 'secureblue:live' into spanish-translation

macaron-jpg committed:
Update audit_secureblue.po

-New translations

macaron-jpg committed:
Update audit_secureblue.po

-fixed pot creation date & .py lines

macaron-jpg committed:
Update audit_secureblue.po

-fixed terms
